index.html

```diff
@@ -6,19 +6,47 @@
     	<meta name="keywords" content="one,two,three">
 
 		<title>Blank Template</title>
+		<link rel="stylesheet" href="css/style.css" />
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
 
 		<!-- CSS links -->
 
 	</head>
-	<body>
+	<body id="container">
+		<header>
+			<h1 class="logo">Relaxr</h1>
+			<h2>Get piece of mind with a single tap</h2>
+			<button>Get it Now</button>
+		</header>
+
+		<section>
+			<h3>Benefits</h3>
+				<p>The pefect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
```

Comment:
Try to keep your indentation consistent. All elements that are at the same depth should have the same amount of indentation. Therefore, your `<h3>` should be indented the same as your `<p>` and `<ul>`